src/extension.ts

```diff
@@ -101,7 +102,7 @@ function removeRegistry(serverUrl: string = REGISTRY_REDHAT_IO): void {
 // TODO: add listRegistries to registry API to allow search by
 // registry URL
 function isRedHatRegistryConfigured(): boolean {
-  const homeFolderPath = process.env.HOME ? process.env.HOME : process.env.USER_PROFILE;
+  const homeFolderPath = homedir();
```

Comment:
looks like it's always returning something so i'm not sure about the next condition if not defined

Reply:
removed check for home dir not being set.